src/polymer-cli.ts

```diff
@@ -183,8 +195,11 @@ export class PolymerCli {
     const commandOptions = parseCLIArgs(commandOptionsRaw);
     logger.debug(`command options parsed from args:`, commandOptions);
 
-    const mergedConfigOptions =
-        Object.assign({}, this.defaultConfigOptions, commandOptions);
+    const mergedConfigOptions = Object.assign(
```

Comment:
Use Object spread syntax:

```ts
const mergedConfigOptions = {
  ...this.defaultConfigOptions,
  ...objectDashToCamelCase(commandOptions),
};
```

In fact, I think (but not 100% sure) object spread support iterables of entries:

```ts
const mergedConfigOptions = {
  ...this.defaultConfigOptions,
  ...Object.entries(commandOptions).map(([k, v]) => [dashToCamelCase(k), v]),
};
```

Reply:
I changed this to the first option because the second ends up spreading the indexes of the map output and adding the reduce seemed to make this line too verbose.